csm_web/scheduler/migrations/0033_rename_subscribe_user_subscribed.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.15 on 2022-11-28 07:29
```

Comment:
You can probably merge all of these migrations together (delete the migration files, and make migrations again), since they're just iterations of the subscribe functionality.